.github/workflows/windows.yml

```diff
@@ -34,16 +34,21 @@ jobs:
       if: steps.cache-llvm.outputs.cache-hit != 'true'
       run: choco install llvm
     - name: Add clang-cl to PATH
-      run: echo "::add-path::C:\Program Files\LLVM\bin"
+      run: echo "::add-path::$Env:ProgramFiles\LLVM\bin"
```

Comment:
What is this `$Env` exactly? Will it always be an alias to `C:\` or something like that?

Reply:
I assume that $Env is an object-like variable accessible from powershell